### PR TITLE
unify extensions into one

### PR DESCRIPTION
**Pull request by ivy-lmu**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -7,20 +7,13 @@
...
-        "${workspaceFolder}/vscode-extensions/config-editor/extension/lib/**/*.js",
-        "${workspaceFolder}/vscode-extensions/inscription/extension/lib/**/*.js",
-        "${workspaceFolder}/vscode-extensions/process-editor/extension/dist/**/*.js",
-        "${workspaceFolder}/vscode-extensions/engine/extension/dist/**/*.js",
-        "${workspaceFolder}/vscode-extensions/project-explorer/extension/dist/**/*.js"
+        "${workspaceFolder}/dist/**/*.js",
       ],
-      "sourceMaps": true
+      "sourceMaps": true,
+      "preLaunchTask": "clean & build node"
```
Comment (ivy-lmu) [inline .vscode/launch.json:16]: before each start the node part is built quickly 🤩

**Review by ivy-cst — APPROVED** [review]
Nice one :+1:   Please @ivy-lli  look closer ;)!
Just css and scripts looks strange in top level, but this was already like this before the change.
image

File: src/base/commands.ts
Comment (ivy-lli) [inline src/base/commands.ts:1]: we could help our devs here with a little bit of ts magic, e.g. by provide a union with all known cammands and let the editor provide an autocomplete for it (https://tsplay.dev/wjp0Ym)

**Review by ivy-lli — APPROVED** [review]
Looks much cleaner in my eyes 👍

File: src/base/commands.ts
Comment (ivy-lmu) [reply src/base/commands.ts:1]: thanks for the hint, will be considered in a followup

**Pull request merged**